website/content/en/docs/kubebuilder-migration/project_migration_guide.md

```diff
@@ -0,0 +1,275 @@
+# Project Migration - Single API version operator
+
+This guide walks through the steps required to migrate an operator from an Operator SDK project layout to a Kubebuilder project layout.
+
+The document considers [memcached operator][memcached-operator] as an example to describe the process of migrating an operator built using Operator SDK to a runnable Kubebuilder project.
+
+## Prerequisites
+- [git][git_tool].
+- [go][go_tool] version 17.03+.
+- [kubectl][kubectl_tool] version v1.11.3+.
+- [kustomize][kustomize_tool] v3.1.0+.
+- Access to a Kubernetes v1.11.3+ cluster.
+
+## Install Kubebuilder
+
+Follow the installation steps provided in the [kubebuilder book][kubebuilder_install] to install kubebuilder.
+
+## Create a new project
+
```

Comment:
Will we not use the @estroz tool to do the migration? So should not its steps change to work with?

Reply:
The tool is still a WIP and needs some input before we can write docs for it. For now we should write docs that assume no tool exists. I’ll update relevant docs once the tool is approved.

Reply:
@camilamacedo86 That tool doesn't exist yet plus I think it still needs some discussion to finalize the workflow.
So we still need this doc for steps in absence of the tool.
And once we are satisfied with the tooling we can update this doc. I expect there will still be manual steps needed even with the tool.